generic/build-arm-none-eabi-gcc.sh

```diff
@@ -0,0 +1,426 @@
+#!/usr/bin/bash
```

Comment:
Can we change to `#!/usr/bin/env bash` for Linux<->OSX compatibility? There's no `/usr/bin/bash` on OSX